lib/scarpe/para.rb

```diff
@@ -1,13 +1,25 @@
 class Scarpe
   class Para
-    def initialize(app, text)
+    def initialize(app, text, stroke: nil, **html_attributes)
       @app = app
       @text = Array(text)
+      @stroke = stroke
+      @html_attributes = Hash(html_attributes)
```

Comment:
I guess we could sanitize these attributes at some point in the future if we so wish?